tests/test_device_validator.py

```diff
@@ -215,6 +215,12 @@ def test_market_maker_setting(self):
         self.assertIsNone(
             validate_market_maker(energy_rate_profile=str({"0": "30", "2": "33"}),
                                   energy_rate_profile_uuid=uuid4()))
+        self.assertIsNone(
+            validate_market_maker(energy_rate_profile=str({"0": "30", "2": "33"}),
+                                  energy_rate_profile_uuid=uuid4(),
+                                  buying_rate_profile=str({"0": "29", "2": "28"}),
+                                  buying_rate_profile_uuid=uuid4(),
```

Comment:
Minor comment here. AFAIR, the d3a-web will not provide a uuid, but a string (containing a uuid).

Reply:
Thanks for pointing it out, you are right, will refine the validation even more